pkg/datamover/datamover.go

```diff
@@ -137,6 +138,12 @@ func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapsho
 	tempVSClassName := fmt.Sprintf("%s-snapclass", backupName)
```

Comment:
Maybe add a little comment about what the tempVSClass ? Not a dealbreaker.

Reply:
There's a small comment about temp vsclass in the backup controller